net/ip_test.go

```diff
@@ -412,6 +412,31 @@ func TestPreviousIP(t *testing.T) {
 	}
 }
 
+func TestNetworkCovers(t *testing.T) {
+	cases := []struct {
+		network string
+		covers  string
+		result  bool
+		name    string
+	}{
+		{"10.0.0.0/24", "10.1.0.0", true, "contains"},
```

Comment:
How is this test case passing?

Reply:
I accidentally pushed the wrong patch to my tree; wasn't expecting it to get picked up :-).  The correct patch is there now.